homeassistant/components/mqtt/client.py

```diff
@@ -377,21 +376,18 @@ async def async_cleanup(self) -> None:
 class MQTT:
     """Home Assistant MQTT client."""
 
-    _mqttc: mqtt.Client
+    _mqttc: mqtt.Client = None  # type: ignore[assignment]
```

Comment:
I think it's also OK to just do like this:
```
_mqttc: mqtt.Client
```

Any attempt to access the attribute before it's set by init_client will then fail.